dom.bs

```diff
@@ -4286,24 +4286,27 @@ empty string instead, and then do as described below, switching on the interface
  <dd><p>Do nothing.
 </dl>
 
-<p>The <dfn attribute for=Node><code>textContent</code></dfn> getter steps are to return the
-following, switching on the interface <a>this</a> <a>implements</a>:
+<p>To <dfn export>get text content</dfn> with a <a for=/>node</a> <var>node</var>,
+then return the following, switching on the interface <var>node</var> <a>implements</a>:
```

Comment:
I'm not entirely convinced "then" works here or in the "set text content" introduction. Let's drop it?

Reply:
Yeah you're right it might (or might not) be grammatically correct, but it almost definitely isn't neccessary. Will drop it.